packages/block-editor/src/components/block-styles/preview.native.js

```diff
@@ -71,7 +71,7 @@ function StylePreview( { onPress, isActive, style, url } ) {
 			return (
 				<Animated.View
 					style={ [ outlineStyle, { opacity }, styles[ name ] ] }
-					key={ outlineStyle }
+					key={ outlineStyle.borderColor }
```

Comment:
This is for fixing the `[object Object]` key error? I'm wondering if we are sure that the `bordercolor` is/will be always unique?

Reply:
In theory it should be unique since there are two borders, one for marking selection (blue) and the second related to masking background (#fff/#000).

Reply:
However I see it's kinda fragile...